index.src.html

```diff
@@ -905,25 +905,37 @@ <h3 id=interface-reporting-observer>Interface {{ReportingObserver}}</h3>
 
   Each {{ReportingObserver}} object has these associated concepts:
     - A <dfn for=ReportingObserver>callback</dfn> function set on creation.
-    - A {{ReportingObserverOptions}} dictionary called
-      <dfn for=ReportingObserver>options</dfn>.
+    - A list of strings called <dfn for=ReportingObserver>types</dfn>.
+    - A boolean called <dfn for=ReportingObserver>buffered</dfn>.
     - A list of {{Report}} objects called the <dfn for=ReportingObserver>report
       list</dfn>, which is initially empty.
 
   A {{ReportList}} represents a sequence of {{Report}}s, providing developers
   with all the convenience methods found on JavaScript arrays.
 
   The <dfn constructor for=ReportingObserver><code>
-  ReportingObserver(|callback|, |options|)</code></dfn> constructor, when invoked, must run
-  these steps:
+  ReportingObserver(|callback|, |options|)</code></dfn> constructor, when
+  invoked, must run these steps:
 
   1. Create a new {{ReportingObserver}} object |observer|.
 
   2. Set |observer|'s <a>callback</a> to |callback|.
 
-  3. Set |observer|'s <a>options</a> to |options|.
+  3. If |options| is omitted:
 
-  4. Return |observer|.
+     1. Set |observer|'s <a>types</a> to an empty list.
```

Comment:
Do we need this here? I believe we define default values in the processing section?